packages/services/f-services/src/validations.js

```diff
@@ -46,8 +46,10 @@ const isValidPostcode = (postcode, locale) => POSTCODE_REGEX[locale].test(postco
 const meetsCharacterValidationRules = value => /^[\u0060\u00C0-\u00F6\u00F8-\u017Fa-zA-Z-' ]*$/.test(value);
 
 const PHONE_REGEX = {
-    'en-GB': /^(\+(44))?[0-9]{10,11}$/,
-    'es-ES': /^\d{9,}$/
+    'en-GB': /^(\+(44))?[0-9]{10}|(\+(44))?[0-9]{11}$/,
```

Comment:
Small thing – as we've extended the GB validation, do we need to cover that change in the tests? (as guessing it wasn't working for some numbers previously?)